index.ts

```diff
@@ -220,7 +220,7 @@ const bits2int_modN = (bytes: Uint8Array): bigint => { // int2octets can't be us
 const i2o = (num: bigint): Bytes => n2b(num);           // int to octets
 declare const globalThis: Record<string, any> | undefined;    // Typescript global symbol available in
```

Comment:
This is dangerous from a type checking standpoint.  It is saying that any undefined variable is available anywhere and should not be type checked at all.

A more type safe alternative would be:
```ts
declare module globalThis {
  const crypto: {
    getRandomValues<T extends Uint8Array>(array: T): T;
    subtle: {
      digest(algorithm: 'SHA-256', data: Uint8Array): Promise<ArrayBuffer>;
      importKey(
        format: 'raw',
        keyData: Uint8Array,
        algorithm: { name: 'HMAC'; hash: { name: 'SHA-256' } },
        extractable: boolean,
        keyUsages: 'sign'[]
      ): Promise<CryptoKey>;
      sign(algorithm: 'HMAC', key: CryptoKey, data: Uint8Array): Promise<ArrayBuffer>;
    };
  };
}
```